nym-vpn-core/crates/nym-vpn-lib-types/Cargo.toml

```diff
@@ -32,3 +32,4 @@ nym-gateway-directory = { workspace = true, optional = true }
 nym-statistics-common = { workspace = true, optional = true }
 nym-vpn-api-client = { workspace = true, optional = true }
 nym-wg-gateway-client = { workspace = true, optional = true }
+strum_macros = "0.26.4"
```

Comment:
strum is already provided at the workspace level, why not using `{ workspace = true }`?